boards/nano_rp2040_connect/Makefile

```diff
@@ -0,0 +1,55 @@
+# Makefile for building the tock kernel for the Arduino Nano RP2040 Connect board.
+
+TOCK_ARCH=cortex-m0p
+TARGET=thumbv6m-none-eabi
+PLATFORM=nano_rp2040_connect
+
+include ../Makefile.common
+
+OPENOCD=openocd
+OPENOCD_OPTIONS=-f openocd.cfg
+
+KERNEL=$(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+KERNEL_WITH_APP=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/release/$(PLATFORM)-app.elf
+
+BOOTSEL_FOLDER=/media/$(USER)/RP2040
+
+
+# Default target for installing the kernel.
+.PHONY: install
+install: flash
+
+.PHONY: flash-debug
+flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
+	elf2uf2 $< $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).uf2
+	if [ -d $(BOOTSEL_FOLDER) ];
+	then
+		cp $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM)-app.uf2 $(BOOTSEL_FOLDER)"
+	else
+		@echo Please edit the BOOTSEL_FOLDER variable to point to you Nano RP2040 Flash Drive Folder
+	fi
+
+.PHONY: flash
+flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+	elf2uf2 $< $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).uf2
+	if [ -d $(BOOTSEL_FOLDER) ];
+	then
+		cp $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM)-app.uf2 $(BOOTSEL_FOLDER)"
+	else
+		@echo Please edit the BOOTSEL_FOLDER variable to point to you Nano RP2040 Flash Drive Folder
+	fi
+
+.PHONY: program
+program: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
```

Comment:
Can you rename this Make-rule to `flash-app` (or something along those lines)? IIRC we settled on -- for the kernel -- "flash" if it's simply writing a binary using some flashing tool and "program" to use `tockloader`. Or am I missing something @bradjc?

Reply:
I think I'm starting to understand why this happens -- I've seen some other makefiles which had been doing this. I believe `program` in this context is confused with a synonym for `application`, i.e. something running on the system instead of the verb (to) program, i.e. writing something to the board.

Reply:
Yep, renamed it.